specifications/api/swagger.yaml

```diff
@@ -232,27 +258,32 @@ x-transactionAmount: &transactionAmount
   <<: *amount
 
 x-transactionInsertedAt: &transactionInsertedAt
-  description: Absolute time the transaction was inserted in a block
+  description: |
+    <span style="position: relative; left: 35px; top: -21px; vertical-align: middle; background-color: rgba(142, 142, 220, 0.05); color: rgba(50, 50, 159, 0.9); margin: 0 5px; padding: 0 5px; border: 1px solid rgba(50, 50, 159, 0.1); line-height: 20px; font-size: 13px; border-radius: 2px;">
+    <strong>if:</strong> status != pending
+    </span><br/>
+    Absolute time the transaction was inserted in a block.
   type: object
   required:
     - block
     - time
   properties:
     time: *date
-    block:
-      type: object
-      required:
-        - slot_number
-        - epoch_number
-      properties:
-        slot_number:
-          type: integer
-          minimum: 0
-          example: 1337
-        epoch_number:
-          type: integer
-          minimum: 0
-          example: 14
+    block: *blockReference
+
+x-transactionPendingSince: &transactionPendingSince
+  description: |
+    <span style="position: relative; left: 35px; top: -21px; vertical-align: middle; background-color: rgba(142, 142, 220, 0.05); color: rgba(50, 50, 159, 0.9); margin: 0 5px; padding: 0 5px; border: 1px solid rgba(50, 50, 159, 0.1); line-height: 20px; font-size: 13px; border-radius: 2px;">
+    <strong>if:</strong> status == pending
+    </span><br/>
+    Absolute time since when the transaction is pending.
```

Comment:
Would recommend "the point in time at which a transaction became pending" as a more natural phrasing here.